fontbe/src/paths.rs

```diff
@@ -65,6 +65,7 @@ impl Paths {
     pub fn target_file(&self, id: &WorkId) -> PathBuf {
         match id {
             WorkId::Features => self.build_dir.join("features.marker"),
+            WorkId::FeaturesAst => self.build_dir.join("features_ast.marker"),
```

Comment:
.marker should mean the file is near-empty. If you are actually writing out the ast in binary it should probably be .bin. Admittedly this is entirely undocumented but you can see it how other files are named.